### DISCUSSION TITLE
Welcome to LazyPrincess Discussions!

### DISCUSSION THREAD

**Discussion by LazyDeveloperr** [LazyDeveloperr/LazyPrincess#57] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LazyDeveloperr** [comment]
What's the error ?

**Reply by Lovelyavimuwal** [reply]
Bro plzz make a link search bot like ye bot only files deta hai aisa bot bnaoo jo links de output me

**Reply by LazyDeveloperr** [reply]
bna hua h waisa bot !! But wo paid h !

**Comment by Zziwadavid51** [comment]
The bot does not respond to requests unless they are from the admin 
Please fix this

**Reply by LazyDeveloperr** [reply]
i think u r missing variable :  LAZY_GROUPS
